### DISCUSSION TITLE
How can I install on MAC Parallels: Windows 11 Pro ARM-based?

### DISCUSSION THREAD

**Discussion by spencer-navy** [posit-dev/positron#5252] [Q&A]
I've tried all options for downloading the different releases but none seem to be compatible. 

This is the error I get for the two Windows-based .exe files:

![image](https://github.com/user-attachments/assets/b4f6abba-c7eb-462d-9cf1-93041192efa2)

.dmg/.deb/.rpm can't open at all. 

Specs in Parallels: 
![image](https://github.com/user-attachments/assets/faac3c06-c926-4f0f-88b8-578ea540aa3f)

**Comment by lionel-** [comment]
Unfortunately we currently don't build Positron for arm64 Windows. I've created https://github.com/posit-dev/positron/issues/5254 to track progress on this.
